bin/git-fork

```diff
@@ -31,28 +31,27 @@ fi
 [ -z "$project" -o -z "$owner" ] && abort "github repo needs to be specified as an argument"
 
 # create fork
-curl -X POST -u "$user" "https://api.github.com/repos/$owner/$project/forks"
+curl_args="-qs -X POST -u $user"
+curl_url="https://api.github.com/repos/$owner/$project/forks"
+RES=$(curl -H "X-GitHub-OTP: $MFA_CODE" $curl_args "$curl_url")
 [ $? = 0 ] || abort "fork failed"
 
 # Check if user has ssh configured with GitHub
-ssh -o BatchMode=yes git@github.com exit >/dev/null 2>&1
-ssh_exit_code=$?
-if [[ "$ssh_exit_code" = 0 ]] || [[ "$ssh_exit_code" = 1 ]]; then
+if ssh -T git@github.com 2>&1 | grep -qi 'success'; then
```

Comment:
Maybe we could tell users what will happen before `ssh`?
Just don't let them surprised.